css/components/content-types/recipe/recipe.css

```diff
@@ -12,3 +12,115 @@
 
   Style guide: components.content_types.recipe
 */
+
+/* TODO: some of this could possibly be components / partials but lets get something to work with that's close */
+
+.page-node-type-recipe .title-main-wrapper {
+  background-color: #ffffff;
+  margin: 0 auto;
+  max-width: 1200px;
+  padding: 2.369em;
+}
+.page-node-type-recipe .layout__region--middle {
+  display: flex;
+  display: -ms-flexbox;
```

Comment:
I think this prefixed version was needed by IE10 which is not suppoerted by Drupal8. Thes supported IE11 version do not need the prefix (as I know).

Reply:
I'll remove this for now. I think we have more problems to solve with Flexbox and IE11 anyway so it makes sense to tackle under a separate issue I suspect.